source/_integrations/velux.markdown

```diff
@@ -52,6 +58,10 @@ Reboots the configured KLF 200 gateway.
 
 In Home Assistant versions up to 2024.12.x, it was recommended to create an automation to reboot the KLF 200 gateway before shutting down or restarting Home Assistant. This automated reboot is now built into the integration, so you no longer need to create a separate automation.
 
+## Removing the integration
+
+This integration follows standard integration removal. No extra steps are required.
+
```

Comment:
```suggestion

{% include integrations/remove_device_service.md %}
```
This text block adds the actual steps: https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/_integration_docs_template.markdown?plain=1#L204